src/components/ReportPage/index.jsx

```diff
@@ -1,8 +1,9 @@
 /* eslint-disable no-return-assign */
 /* eslint-disable no-fallthrough */
 /* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
-import React, { PureComponent } from 'react';
+import React, { PureComponent, Fragment } from 'react';
 import PropTypes from 'prop-types';
+import dotenv from 'dotenv';
```

Comment:
no need for this in React. use `process.env`

Reply:
If I don't include the dotenv  module, when I use process .env, it does not work